src/input/mod.rs

```diff
@@ -2350,7 +2366,8 @@ impl State {
                 &self.common.config,
                 self.common.event_loop_handle.clone(),
             ),
-
+            // NOTE: implementation currently assumes actions that apply to outputs should apply to the active output
+            // rather than the output that has keyboard focus
```

Comment:
Yes, this seems right to me, as `map_window` should still happen on the active_output.

Reply:
Would this comment be useful for other contributors or should I go ahead and remove it?

Reply:
Please feel free to add these kinds of comments that explain design decisions!